addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -71,8 +73,12 @@ export class WebsiteLoader extends Component {
             (isVisible) => {
                 if (isVisible) {
                     window.addEventListener("beforeunload", this.showRefreshConfirmation);
+                    if (!this.state.selectedFeatures || this.state.selectedFeatures.length === 0) {
+                        this.initProgressBar();
+                    }
                 } else {
                     window.removeEventListener("beforeunload", this.showRefreshConfirmation);
+                    clearInterval(this.updateProgressInterval);
                 }
 
                 return () => window.removeEventListener("beforeunload", this.showRefreshConfirmation);
```

Comment:
You see to clearInterval if the loader becomes invisible, should you not also if the effect is removed here?